src/Generators/Ics.php

```diff
@@ -64,14 +69,24 @@ public function generate(Link $link): string
         $url[] = 'END:VEVENT';
         $url[] = 'END:VCALENDAR';
 
-        return $this->buildLink($url);
+        $format = $this->presentationOptions['format'] ?? 'html';
```

Comment:
would be nice to add these as public class constants so we can do `['format' => Ics::FILE]`

Reply:
thank you, done by 28d5971